openlibrary/macros/SearchResultsWork.html

```diff
@@ -1,26 +1,30 @@
 $def with (doc, decorations=None, cta=True, availability=None, extra=None)
 
+$ ocaid = doc.get('availability', {}).get('identifier')
 $ is_work = doc.get('type', {}).get('key') == '/type/work'
 $ book_url = doc.url() if is_work else doc.key
 $ cover_ids =  [cover for cover in doc.get('covers', []) if cover != -1]
 <li class="searchResultItem" itemscope itemtype="https://schema.org/Book">
   <span class="bookcover">
-    $if doc.get('cover_i') or cover_ids:
+    $if ocaid:
+        $# We should use "/b/ia/%s-M.jpg" and make it fetch covers if they're missing
+        $# presently, it frequently misses.
+        $ cover = "//archive.org/download/%s/page/cover_w60_h60.jpg" % ocaid
```

Comment:
:(

Reply:
I think this is a blocker. With this, there's no way for an Open Library editor to correct an edition's cover and have it appear in the search results. It also makes all the covers that librarians have added in the past inaccessible from the search results.

Reply:
Proposal? At this point in the code, we only have an `ocaid`; we only use this strategy if we have the book. Granted Archive.org can be wrong.

Reply:
I was able to find a compromise where the average case will be that the Archive.org availability API returns an edition olid back to us and we can use this to fetch the correct edition. So we only fallback to archive.org when there's no `olid` in the availability. Once the archive.org item is updated with an olid, the cover will show us (and librarians have the ability to fix that with the sync button on the books page)